.eslintrc

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
You don't need this. There's already an `.eslintrc.json` file in the root that extends the config that I use on all my projects.